CHANGES.rst

```diff
@@ -4,6 +4,18 @@ Changes
 0.12.1 (unreleased)
 ~~~~~~~~~~~~~~~~~~~
 
+- Add a script to check a database to see if a class is used as a
+  persistent object in it or not or if it is missing.
+- Various improvements on the previous script looking for broken
+  records and references.
```

Comment:
When feature X is added and then improved between two consecutive releases, it's best to collapse the changelog mentions into just "feature X added", IMHO.